src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/EventPipeProvider.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Diagnostics.NETCore.Client
 {
     public sealed class EventPipeProvider
     {
-        public EventPipeProvider(string name, EventLevel eventLevel, long keywords = 0, IDictionary<string, string> arguments = null)
+        public EventPipeProvider(string name, EventLevel eventLevel, long keywords = 0xF00000000000, IDictionary<string, string> arguments = null)
```

Comment:
Out of curiosity, what does `0xF00000000000` represent?

Reply:
that should map to https://github.com/dotnet/diagnostics/blob/master/documentation/dotnet-trace-instructions.md, right? I always used Fs to get everything.

Reply:
This is keyword that gets appended to Events that are written without any keywords specified. 

For example you won't be able to listen to the events written in the following EventSource by default.

```
    public class MyEventSource : EventSource
    {
        public static MyEventSource Log = new MyEventSource();

        public void Event1() { WriteEvent(1); }
        public void Event2(string fileName) { WriteEvent(2, fileName); }
        public void Event3() { WriteEvent(3); }
    }
``` 

With this change the simplest set of EventPipeProvider is just `new EventPipeProvider("MyEventSource", EventLevel.Informational)`. 

This is also the default behavior of EventListener.EnableEvents() which is what this API was meant to correspond with.